go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/konveyor/forklift-controller
 go 1.14
 
 require (
+	github.com/99designs/gqlgen v0.14.0
```

Comment:
This isn't a code generator?

Reply:
Oh I see, that's actually both, the GraphQL library which uses a generator.

Reply:
@jortel, gqlgen is used here: https://github.com/konveyor/forklift-controller/pull/358/files#diff-72eef241725fe361fe8d4e1e9bbd25b25915aa2ae9085c80175da93856bcdbb6R15-R16